{{cookiecutter.project_name}}/.github/workflows/deploy-docs.yaml

```diff
@@ -28,5 +28,5 @@ jobs:
         run: |
           mkdir -p docs
           touch docs/.nojekyll
-          poetry run gen-doc -d docs src/{{cookiecutter.__project_slug}}/schema/{{cookiecutter.__project_slug}}.yaml
-          poetry run mkdocs gh-deploy
+          make gendoc
```

Comment:
No `poetry run` before the commands?

Reply:
No, `poetry run` is not neccessary because the _make_ targets already have poetry commands prefixed with `$(RUN)` which is a shorthand for `poetry run`.

See the `gendoc` target as an example: https://github.com/linkml/linkml-project-cookiecutter/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/Makefile#L158-L160